AnkiDroid/src/androidTest/java/com/ichi2/anki/tests/libanki/ModelTest.java

```diff
@@ -37,9 +37,11 @@ public void tearDown() {
     public void bigQuery() throws IOException {
         Models models = testCol.getModels();
         Model model = models.all().get(0);
-        StringBuffer buf = new StringBuffer();
-        for (long i = 0L; i < 400_000L; ++i ) {
-            buf.append("abcdefghijkl");
+        final int size = 4 * 1024 * 1024;
+        StringBuilder buf = new StringBuilder(size * 1.01);
+        // * 1.01 for padding
+        for (int i = 0; i < size; ++i ) {
+            buf.append("a");
         }
         model.put("test", buf.toString());
```

Comment:
```suggestion
        model.put(testString, buf.toString());
```

I think this is what David and I meant - make it so that no one has to count anything

Reply:
I was referring to the line: `buf.append( );`, the key shouldn't matter as it's only set once, but there's not a problem with using the same variable